adafruit_matrixkeypad.py

```diff
@@ -62,13 +62,15 @@ def pressed_keys(self):
             pin.direction = Direction.INPUT
             pin.pull = Pull.UP
 
-        for row in range(len(self.row_pins)):
+        for row in range(  # pylint: disable=consider-using-enumerate
```

Comment:
Just wanted to double check:

What is the reason you're using `for row in range(..):` here, and using `for col, val, in enumerate(..):` further ahead on line 72?